rageval/metrics/_answer_claim_recall.py

```diff
@@ -142,21 +137,21 @@ def _compute_batch(
         dataset: datasets.Dataset
     ) -> list:
         """
-        Evaluate the groundedness of a batch of answers.
+        Evaluate the correctness of a batch of answers.
 
-        Firstly,split the answer into a set of claims.
+        Firstly, split the gt_answer into a set of claims. (There are many ways to obtain claims.)
+        It is assumed that claims have been obtained here.
         Then, compute the faithfulness score of each claim. The faithfulness is a binary score.
         Finally, aggregate all faithfulness score of each claim.
         """
 
-        answers, contexts = (
+        answers, claims = (
```

Comment:
In the original paper, claims are induced from gold answers by Instruct-GPT. Thus, this metric rely on two out-side models, including a LLM and a NLI model.